### PR TITLE
Scheduler doesn't support the +30 minutes time-zones (T681124)

### PR DESCRIPTION
**Pull request by andrewmakarov**

### CODE REVIEW

File: js/ui/scheduler/ui.scheduler.subscribes.js [hunk 1]
```diff
@@ -371,7 +371,7 @@ var subscribes = {
             "DATE": function() {
                 var dateTimeFormat = "monthAndDay",
                     startDateString = dateLocalization.format(startDate, dateTimeFormat),
-                    isDurationMoreThanDay = (endDate.getTime() - startDate.getTime()) > 24 * 3600000;
+                    isDurationMoreThanDay = (endDate.getTime() - startDate.getTime()) > 24 * toMs("hour");
```
Comment (DokaRus) [inline js/ui/scheduler/ui.scheduler.subscribes.js]: >24 * toMs("hour");

`toMs("day")`?

File: testing/tests/DevExpress.ui.widgets.scheduler/subscribes.tests.js [hunk 2]
```diff
@@ -832,6 +831,22 @@ QUnit.test("'convertDateByTimezone' should return date according to the custom t
...
+QUnit.test("'convertDateByTimezone' should return date according to the custom timeZone with non-integer number", function(assert) {
+    var timezone = { id: "Australia/Broken_Hill", value: 9.5 };
+    this.createInstance();
+
+    this.instance.option({
+        timeZone: timezone.id
+    });
+
+    var date = new Date(2015, 6, 3, 3),
+        timezoneDifference = date.getTimezoneOffset() * 60000 + timezone.value * 3600000;
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets.scheduler/subscribes.tests.js]: You can use `dateUtils.dateToMilliseconds("minute")`, `dateUtils.dateToMilliseconds("hour")` here

File: js/ui/scheduler/ui.scheduler.subscribes.js [hunk 3]
```diff
@@ -675,50 +675,44 @@ var subscribes = {
 
     convertDateByTimezone: function(date, appointmentTimezone) {
         date = new Date(date);
+        var tzOffsets = this._subscribes.getComplexOffsets(this, date, appointmentTimezone);
```
Comment (lukyanovas) [inline js/ui/scheduler/ui.scheduler.subscribes.js:678]: In common case use context binding instead of passing the instance as an argument

**Review by lukyanovas — APPROVED** [review]
LGTM

**Pull request merged**